### ISSUE TITLE
handleTapGesture needs to be moved to MessageCollectionViewCell

### ISSUE THREAD

**Issue by SD10** [MessageKit/MessageKit#773]
This needs to be moved to the more general `MessageCollectionViewCell` class

https://github.com/MessageKit/MessageKit/blob/69c5414c224a98891e14984e83228740e4339219/Sources/Views/MessagesCollectionView.swift#L78-L87

**Comment by maxxx777** [comment]
Hi @SD10,

I'd like to take the issue, but I need some clarification:

the idea is to remove the method from `MessagesCollectionView` to avoid calculation of gesture location and cell index path:

```
let touchLocation = gesture.location(in: self)
guard let indexPath = indexPathForItem(at: touchLocation) else { return }
```

either, the idea to move `handleTapGesture` method from `MessageContentCell` into `MessageCollectionViewCell` ?

**Comment by nathantannar4** [comment]
Implemented in #950
